assets/script.js

```diff
@@ -124,3 +124,24 @@ function searchCard() {
     }
   }
 }
+
+$(document).ready(function() {
+  $(window).scroll(function() {
+    if ($(this).scrollTop() > 20) {
+      $('#toTopBtn').fadeIn();
+    } else {
+      $('#toTopBtn').fadeOut();
+    }
+  });
+
+  $('#toTopBtn').click(function() {
+    $("html, body").animate({
+      scrollTop: 0
+    }, 1000);
+    return false;
+  });
+});
+menu.onclick = () => {
```

Comment:
This entire block needs to be deleted. `menu`is not defined and it is causing a JS error. And this block is not doing anything here. Remove it please from line 144 to 147.